docs/walkthrough/walkthrough-lighthouse.md

```diff
@@ -0,0 +1,421 @@
+# Tekton Dashboard walkthrough - Lighthouse
+
+This guide walks you through installing a working Tekton Dashboard locally from scratch and adding [Lighthouse](https://github.com/jenkins-x/Lighthouse) on top of it.
+It covers the following topics:
+
+* [Before you begin](#before-you-begin)
+* [What is Lighthouse](#what-is-lighthouse)
+* [Installing a working Tekton Dashboard locally from scratch](#installing-a-working-tekton-dashboard-locally-from-scratch)
+* [Exposing your cluster ingress controller over ngrok](#exposing-your-cluster-ingress-controller-over-ngrok)
+* [Creating a GitHub OAuth token](#creating-a-github-oauth-token)
+* [Preparing your environment](#preparing-your-environment)
+* [Installing Lighthouse](#installing-lighthouse)
+* [Configuring Lighthouse](#configuring-Lighthouse)
+* [Configuring GitHub to send events to Lighthouse](#configuring-github-to-send-events-to-lighthouse)
+* [Configuring Lighthouse jobs](#configuring-lighthouse-jobs)
+* [Lighthouse running jobs when opening a pull request](#lighthouse-running-jobs-when-opening-a-pull-request)
+* [Cleaning up](#cleaning-up)
+
+## Before you begin
+
+Before you begin, make sure the following tools are installed:
+
+1. [`kind`](https://kind.sigs.k8s.io/): For creating a local cluster running on top of docker.
+1. [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/): For interacting with your kubernetes cluster.
+1. [`helm`](https://helm.sh/docs/intro/install/): For installing helm charts in your kubernetes cluster.
+
+You will also need a [GitHub](https://github.com/) account and a [ngrok](https://ngrok.com/) to complete this walkthrough.
+
+## What is Lighthouse
+
+From the [Lighthouse GitHub page](https://github.com/jenkins-x/lighthouse):
+
+> Lighthouse is a lightweight ChatOps based webhook handler which can trigger Jenkins X Pipelines and soon Tekton Pipelines based on webhooks from multiple git providers such as: GitHub, GitHub Enterprise, BitBucket Server, BitBucket Cloud, GitLab, Gitea etc
```

Comment:
`s/and soon Tekton Pipelines/and Tekton Pipelines/`, and I'd remove the `BitBucket Cloud` and `Gitea` from the list - thanks for reminding me that Lighthouse's README needs to be updated... =)

Reply:
fixed, thanks